packages/babel-compiler/babel-compiler.js

```diff
@@ -1,5 +1,6 @@
 var semver = Npm.require("semver");
 var JSON5 = Npm.require("json5");
+var SWC = Npm.require("@swc/core");
```

Comment:
How Meteor packages are published isn't compatible with @swc/core since the published package only has a native module for the architecture and os it is published from, and uses the much slower web assembly build on other architectures.

My zodern:standard-minifier-js package also uses swc, so I created an npm package that fixes this: https://github.com/zodern/meteor-package-install-swc. I can add you as a collaborator to the package.

Reply:
Oh, I see. Please add me as a collaborator. It would be good to keep the SWC version up to date and ensure it's used correctly in the packages when published.

Reply:
@zodern: I added a PR to update SWC to the latest version. If you could approve that and publish a new version, then we can include it on this PR.

https://github.com/zodern/meteor-package-install-swc/pull/1